Shape_detection/examples/Shape_detection/shape_detection_basic_deprecated.cpp

```diff
@@ -2,18 +2,17 @@
 #pragma warning(disable:4244) // boost::number_distance::distance()
                               // converts 64 to 32 bits integers
 #endif
-
 #include <CGAL/internal/disable_deprecation_warnings_and_errors.h>
 
+#include <cassert>
```

Comment:
anything deprecated should go into test

Reply:
This was an example in the first version of this package to show the difference between the original code by Simon and the new generic version. Maybe now it is already a good idea to put it to the test folder.